### DISCUSSION TITLE
tune the consistency level

### DISCUSSION THREAD

**Discussion by 1292253144** [milvus-io/milvus#28005] [Q&A and General discussion]
我认为一致性级别是在进行向量相似性搜索或查询时进行指定的，但是为什么在创建collection时也会指定一致性级别。那么在创建collection指定的一致性级别与查询时指定的会有什么冲突吗？

**Comment by xiaofan-luan** [comment]
请求覆盖表属性，不配置请求参数，配置了表属性就可以不设置请求参数
